carbon-design-system/carbon-website
# chore: update to 10.3

- Update to 10.3 and 7.3 for component package
- Update left nav override styles to be compatible with new shell updates

## Testing

- Make sure left nav render correctly
- Make sure components render correctly

closes #1639
